testing/regress/ecl/childds1.ecl

```diff
@@ -53,6 +53,6 @@ cntBad := COUNT(ds(assertTrue(seq > 10, 'seq > 10'))) + NOFOLD(100000);
 
 //NOFOLD is required to stop code generator converting this to a filter (trueValue && ...) which means that cntBad is evaluated always
 //See childds1err.ecl
-cond := IF(trueValue, ds, NOFOLD(ds)(seq != cntBad));
+cond := IF(trueValue, ds, NOFOLD(ds)(seq != cntBad, hint(testHintIsSupported(1))));
```

Comment:
It doesn't really matter where it goes.  I always run the compiler regression suite on testing/regress, ecl/regress and the private regression suite.  Adding it to the runtime regression suite helps ensure it doesn't have any unwanted side-effects.